src/chainparams.cpp

```diff
@@ -265,6 +266,7 @@ class CUnlParams : public CChainParams
             boost::assign::list_of(0x42)(0x69)(0x67)(0x20).convert_to_container<std::vector<unsigned char> >();
```

Comment:
why there's a different way to set up `EXT_PUBLIC_KEY` and `EXT_SECRET_KEY` in respect to what we do for bch testnet a few lines below (L368)?

Reply:
It's an older definition..the ones carried over from ABC were newer and look better , but there's nothing wrong with what is there.  Maybe in a cleanup PR we can update the old definitions so everything is uniform.